Mathlib/Combinatorics/Enumerative/Bell.lean

```diff
@@ -168,4 +175,27 @@ theorem uniformBell_eq_div (m : ℕ) {n : ℕ} (hn : n ≠ 0) :
   · exact Nat.mul_pos (Nat.pow_pos (Nat.factorial_pos n)) m.factorial_pos
   · rw [← mul_assoc, ← uniformBell_mul_eq _ hn]
 
+/-- The `n`th standard Bell number,
+which counts the number of partitions of a set of cardinality `n`. -/
+def standardBell : ℕ → ℕ
```

Comment:
I would call it `Nat.bell`. There could be no confusion with the other one which has `Multiset` in its namespace, and maybe both should be `protected`.